_posts/2019-10-04-Siddhant.md

```diff
@@ -0,0 +1,9 @@
+
+---
+name: "Siddhant Singh"
+github_id: "siddhantsingh186"
+---
+
+Welcome to "Introduction to Git and Hacktoberfest Study Jam!"
```

Comment:
I don't think you need this line 🤔  Remove it and commit, so that I can merge this PR soon.